stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -25,6 +25,14 @@ import io.getstream.chat.android.client.api.models.QueryChannelsRequest
 import io.getstream.chat.android.client.api.models.QueryUsersRequest
 import io.getstream.chat.android.client.api.models.SearchMessagesRequest
 import io.getstream.chat.android.client.api.models.querysort.QuerySorter
+import io.getstream.chat.android.client.api2.endpoint.ChannelApi
+import io.getstream.chat.android.client.api2.endpoint.ConfigApi
+import io.getstream.chat.android.client.api2.endpoint.DeviceApi
+import io.getstream.chat.android.client.api2.endpoint.GeneralApi
+import io.getstream.chat.android.client.api2.endpoint.GuestApi
+import io.getstream.chat.android.client.api2.endpoint.MessageApi
+import io.getstream.chat.android.client.api2.endpoint.ModerationApi
+import io.getstream.chat.android.client.api2.endpoint.UserApi
```

Comment:
Hey @leandroBorgesFerreira,

Not sure how these package changes are related to R8 full mode support? :)

Reply:
Seems to me like this is for easier application of the keep rules, so that anything serialized is not obfuscated by R8.

Reply:
This is necessary to use this rule: 
```
-keep class io.getstream.chat.android.client.api2.endpoint.** { *; }
```
instead of this one:
```
-keep class io.getstream.chat.android.client.api2.** { *; }
```
Because it would make the rule too broad and include many classes that are not necessary. Although we could simple include class by class, this would be harder to maintain in the future, because we would need to add a new rule for every new classes added in this package.